### DISCUSSION TITLE
Pull Requests

### DISCUSSION THREAD

**Discussion by halloworldconnect** [community/community#146759] [Pull Requests]

**Comment by MindfulLearner** [comment]
To pull request you should fork the repository, make your own branch push then you will see a new button compare and pull request! 

if this was useful to you dont forget to give an accepted answer ty!

**Reply by Kobi-Blade** [reply]
Not useful, pull requests are not working when we trying to pull changes up to specific commits, instead of all the changes.

Only workaround is to create another branch on your fork, to pull the changes up to the specific commit you want.
